src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -2141,15 +2142,7 @@ mod tests {
             }
 
             fn autobrake_mode(&mut self) -> AutobrakeMode {
-                let mode = self.simulation_test_bed.read_f64("AUTOBRAKES_ARMED_MODE") as u8;
-
-                match mode {
-                    0 => AutobrakeMode::NONE,
-                    1 => AutobrakeMode::LOW,
-                    2 => AutobrakeMode::MED,
-                    3 => AutobrakeMode::MAX,
-                    _ => AutobrakeMode::NONE,
-                }
+                AutobrakeMode::from(self.read_f64("AUTOBRAKES_ARMED_MODE") as u8)
```

Comment:
```suggestion
                self.read_f64("AUTOBRAKES_ARMED_MODE").into()
```
and:
```rust
impl From<f64> for AutobrakeMode {
    fn from(value: f64) -> Self {
        match value as u8 {
            0 => AutobrakeMode::NONE,
            1 => AutobrakeMode::LOW,
            2 => AutobrakeMode::MED,
            3 => AutobrakeMode::MAX,
            4 => AutobrakeMode::HIGH,
            5 => AutobrakeMode::RTO,
            6 => AutobrakeMode::BTV,
            _ => AutobrakeMode::NONE,
        }
    }
}
```

Note that there are no u8 `From` usages, so `From<u8>` can be removed.

Reply:
Thanks, really need to get used to the enum rust spirit :D